automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/DuosApiSpec.scala

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsde.test.api.orch
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.service.Orchestration
+import org.scalatest.{FreeSpec, Matchers}
+import spray.json.DefaultJsonProtocol._
+import spray.json.{JsObject, JsonParser}
+import spray.json.lenses.JsonLenses._
+
+class DuosApiSpec extends FreeSpec with Matchers {
+
+  "/duos/researchPurposeQuery" - {
+    "should return empty filter when all parameters are defaulted" in {
+      implicit val authToken: AuthToken = UserPool.chooseStudent.makeAuthToken()
+      val result = Orchestration.duos.researchPurposeQuery()
+      val json = JsonParser(result)
+
+      json.extract[JsObject]('bool / 'must.?) shouldBe empty
+      json.extract[JsObject]('bool / 'should.?) shouldBe empty
```

Comment:
Should this maybe also check the presence of `disable_coord`, `adjust_pure_negative` and `boost` fields?

Reply:
I have no idea what those properties are for, and it's not clear to me that there's anything in our code that is causing those to be set any particular way. @ahaessly, do you know anything about those?

Reply:
no i don't know what those do. they are added on by default when ES generates the query. i think it's ok not to check them. they might change with a different version of ES